pyproject.toml

```diff
@@ -25,7 +25,7 @@ python-dateutil = "^2.8.2"
 requests = "^2.25.0"
 cryptography = ">=3.4.8"
 tqdm = "^4.32.1"
-pydantic = ">=1.9.0"
+pydantic = ">=1.10.14, <2.0"
```

Comment:
Had to upgrade pydantic because otherwise I'm hitting this issue: https://github.com/pydantic/pydantic/issues/5821 when using the `Literal`

Reply:
Ah, yes! Fixed.